### PR TITLE
[node] Clean up dependency of tfc and tfl in node tests

### PR DESCRIPTION
**Pull request by kangyizhang**
Removed unnecessary dependency of tfjs-core and tfjs-layers. Direct import of tfjs-core is still required for backend configuration and test. Direct import of tfjs-layers is fully removed.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/2995)

### CODE REVIEW

File: tfjs-core/yarn.lock [hunk 1]
```diff
@@ -18,29 +18,10 @@
...
-
-"@bazel/bazel@^0.24.0":
-  version "0.24.0"
-  resolved "https://registry.yarnpkg.com/@bazel/bazel/-/bazel-0.24.0.tgz#f4e68e3680ac299858c24c26be3d08d1151e78fc"
-  integrity sha512-/5E55tqH9ogAGF9Dd7RSCJmk7/xdlsPTAhsX3yEsEMs7GLdHlgD3jbeePsKUiHKKr8LXAufjTs2pXQfjrkZRMg==
-  optionalDependencies:
-    "@bazel/bazel-darwin_x64" "0.24.0"
-    "@bazel/bazel-linux_x64" "0.24.0"
-    "@bazel/bazel-win32_x64" "0.24.0"
+"@bazel/bazelisk@^1.3.0":
```
Comment (lina128) [inline tfjs-core/yarn.lock:21]: Hi Kangyi, is this change intended?

Comment (kangyizhang) [reply tfjs-core/yarn.lock:21]: Yes, this is an update based on https://github.com/tensorflow/tfjs/pull/2912

Comment (lina128) [reply tfjs-core/yarn.lock:21]: Got it. Thanks!
